src/styles/blocks_main/_main-container.scss

```diff
@@ -0,0 +1,11 @@
+@import "variables";
+
+.main_container {
+  display: flex;
+  justify-content: center;
+  flex-wrap: wrap;
+  box-sizing: border-box;
+  max-width: 1024px;
```

Comment:
don't just keep it a `magic number` - provide a calculation
introduce additional variables if needed, e.g. `4 * $card-width + 3 $card-spacing`
and use absolute `width` instead `max-width`

Reply:
`max-width` causes this:
![max-width](https://user-images.githubusercontent.com/39308717/160996722-55fcfa07-4a3a-4ef2-82a5-f4a5151b86e4.png)